doc/api/globals.md

```diff
@@ -637,6 +637,73 @@ logical processors available to the current Node.js instance.
 console.log(`This process is running on ${navigator.hardwareConcurrency} logical processors`);
 ```
 
+### `navigator.language`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {string}
+
+The `navigator.language` read-only property returns a string representing the
+preferred language of the Node.js instance.
+
+The value is representing the language version as defined in [RFC 5646][].
+
+The value of `navigator.language` can set via the `LC_ALL` environment
```

Comment:
If this doesn't work in Windows, we should mention what the process is there.

Reply:
`LC_ALL` is also not the only way to set it in Unix. The ICU function that gets the value is a bit complex and depends on the system.

Reply:
So what should we recommend? `Object.defineProperty`?

Reply:
We could say for *nix systems  use LC_ALL and generally you could overwrite it with Object.defineProperty true.

Reply:
If there’s a solution that works everywhere, we should just say that one and leave out platform-specific alternatives.

Reply:
Related: https://github.com/nodejs/node/issues/28099. It seems like Node doesn’t actually provide a cross-platform way to set the default locale at the moment.

I have a feeling the only certain way to do this is `Object.defineProperty` to override `language`/`languages` directly. But that’s probably not a great thing to recommend as people might assume it would also be configuring Node’s other internationalization-related features, like to change how `toLocaleString` behaves.

I think for the purposes of this PR maybe let’s just remove any mention of how to set this value? Leave the `LC_ALL` stuff in the tests, but the docs can just present this as a read-only property that always returns the value of `Intl.DateTimeFormat().resolvedOptions().locale` (or whatever the correct equivalence is) and it’s a separate issue that the `Intl` APIs need to provide a universal way to set the locale.

Reply:
@GeoffreyBooth PTAL at #50538

Reply:
I think if you make this dependent on #50538, this could take a _long_ time to land. Why not just update the docs so that they’re correct for all platforms and then let’s land this? And if/when we add a cross-platform way to set the locale, the `navigator.language` docs can be updated accordingly.

Reply:
Adapted. PTAL